README.md

```diff
@@ -1,3 +1,4 @@
+[![Coverage Status](https://coveralls.io/repos/github/andela/octopus-bn-backend/badge.svg?branch=ch-configure-coveralls-170947427)](https://coveralls.io/github/andela/octopus-bn-backend?branch=ch-configure-coveralls-170947427)
 Barefoot Nomad - Making company travel and accomodation easy and convinient.
 =======
 [![](https://img.shields.io/badge/Reviewed_by-Hound-a873d1.svg)](https://houndci.com)
```

Comment:
@hakoemmy can these badges be arranged in order because As I can see one badge is above the title and another is below